nautobot/extras/api/serializers.py

```diff
@@ -99,10 +100,48 @@
 )
 
 #
-# Mixins
+# Mixins and Base Classes
 #
 
 
+class NautobotModelSerializer(RelationshipModelSerializerMixin, CustomFieldModelSerializer, ValidatedModelSerializer):
```

Comment:
Since `RelationshipModelSerializerMixin` extends `ValidatedModelSerializer` do we need to include `ValidatedModelSerializer` here?

Reply:
The fact that the mixins are actually full-featured serializers in their own right is something I consider an implementation detail. I think it's better to be explicit as to the base class here that we're augmenting with mixins, even if it is a bit redundant.